packages/plugin-serverless/src/commands/serverless/env/get.js

```diff
@@ -0,0 +1,41 @@
+const { TwilioClientCommand } = require('@twilio/cli-core').baseCommands;
+
+const {
+  handler,
+  cliInfo,
+  describe,
+} = require('twilio-run/dist/commands/env/env-get');
+const {
+  convertYargsOptionsToOclifFlags,
+  normalizeFlags,
+  createExternalCliOptions,
+  getRegionAndEdge,
+} = require('../../../utils');
+
+const { flags, aliasMap } = convertYargsOptionsToOclifFlags(cliInfo.options);
+
+class EnvGet extends TwilioClientCommand {
+  async run() {
+    await super.run();
+
+    let { flags, args } = this.parse(EnvGet);
```

Comment:
You pointed out how I was using `this.args` and `this.flags` in my last PR. It turns out that is because the [`BaseCommand` does the arg/flag parsing already](https://github.com/twilio/twilio-cli-core/blob/main/src/base-commands/base-command.js#L35-L37). So you don't need to do it again and can just use `this.flags` and `this.args` too.